src/QMCHamiltonians/tests/test_coulomb_pbcAA.cpp

```diff
@@ -59,17 +60,23 @@ TEST_CASE("Coulomb PBC A-A", "[hamiltonian]")
   REQUIRE(consts == Approx(-3.1151210154));
 
   double val = caa.evaluate(ions);
-  //cout << "val = " << val << std::endl;
-  REQUIRE(val == Approx(-1.418648723)); // not validated
+  //std::cout << "val = " << val << std::endl;
+  REQUIRE(val == Approx(vmad_sc));
```

Comment:
REQUIRE stops the execution. So better suited for flow control or nullptr checks.
CHECK are more suitable for value checks. So please use CHECK.